common/copy_component_versions_file.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
What's the benefit of copying to an intermediate file which is $IMAGE_DIR/component_version.txt rather than this final destination file?